### ISSUE TITLE
Missing accessibility slider gestures

### ISSUE THREAD

**Issue by defagos** [SRGSSR/srgmediaplayer-apple#116]
As noted by @NicolasBrunnerSRF the accessibility slider increment / decrement gestures do not work properly. The slider namely reflects the current playback position and therefore the incremented / decremented position is immediately reset.

### Issue type

Incorrect behavior

### Environment information

* Library version: 6.2.2
* iOS / tvOS version: Any
* Device: Any

### Reproducibility

Always reproducible

### Steps to reproduce

1. Run the demo.
1. Enable VoiceOver and play some on-demand example.
1. Select the slider using VoiceOver and swipe up (increment) or down (decrement). The playback position does not change.

**Comment by defagos** [comment]
Fixed on the `feature/accessibility-slider-navigation` branch.

**Comment by pyby** [comment]
Default 15 seconds like the AVPlayer is a good idea.
